public/src/feed/channel-view.html

```diff
@@ -69,6 +74,9 @@ <h2>Cards by
       <div id="noItems" class="hidden">There are no matching cards in this category.</div>
       <card-feed id="feed" items="[[items]]" on-card-open-request="_onCardOpenRequest" on-card-closed="_onCardClosed" on-impression="_onCardImpression"
         on-card-payment-due="_onCardPay"></card-feed>
+      <div class="buttons">
+        <button id="btnMore" style="display:none;" on-click="_onMoreClick">More</button>
+      </div>
     </div>
```

Comment:
I think we should make this  like an infinite scroll, instead of a more button. 
It's fine for now. 
Also, it's better to hide "buttons" not just the button because buttons adds margin/padding. Don't think it matters here but this kind stuff usually messes up scrolling - i.e. extra scroll when there shouldn't be

Reply:
Also not that this more button is above the "promoted" cards. Not sure if this was intentional on the channel page.